### PR TITLE
Change due to proj6

### PR DESCRIPTION
**Pull request by rmarzocchi84**
Add instructions to add a line to proj.db file including the NtV2 file

### CODE REVIEW

File: docs/user_manual/working_with_projections/working_with_projections.rst [hunk 1]
```diff
@@ -286,19 +286,47 @@ Integrate an NTv2-transformation in QGIS
...
+    
+    .. _figure_nadgrids:
+   
    .. figure:: img/nadgrids_example.png
       :align: center
-
+      
       Setting an NTv2 transformation
+      
+#. Step 3. Since QGIS 3.10.X it is also needed to add a line for each transformation in
```
[the remark this answers is not in the document]
Comment (nyalldawson) [reply docs/user_manual/working_with_projections/working_with_projections.rst:302]: I don't like this inclusion -- it's definitely NOT a recommended approach to follow, nor should the vast majority of users ever need to do this. But the wording of this section makes it sound like anyone who wants a ntv2 transformation must do these steps.

Rather, the recommended approach is to let proj and QGIS handle all this for you, and not touch ANY of these files yourself! If proj doesn't know about a particular grid shift file, then the recommended approach is to follow https://github.com/OSGeo/PROJ-data/blob/master/CONTRIBUTING.md . The described approach is an absolute last effort which is NOT recommended, isn't necessarily stable, and probably belongs better as a stack exchange answer or a blog post vs including in the official docs...

[the remark this answers is not in the document]
Comment (rmarzocchi84) [reply docs/user_manual/working_with_projections/working_with_projections.rst:302]: Effectively it is a complicated approach, but for some countries (e.g. Italy) it is not possible to include NTv2 transformation in proj library due to license of NTv2 files. I am absolutely contrary to the choose of Italian governement but I cannot change it (the GFOSS.it group has been working for change for many years)

**Comment by rmarzocchi84** [comment]
@DelazJ thanks for your changes!

File: docs/user_manual/working_with_projections/working_with_projections.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (rmarzocchi84) [reply docs/user_manual/working_with_projections/working_with_projections.rst:302]: Perhaps we can add as introduction your explication:

_If proj doesn't know about a particular grid shift file, then the recommended approach is to follow https://github.com/OSGeo/PROJ-data/blob/master/CONTRIBUTING.md. The approach described in the following section is an absolute last effort which is NOT recommended if not strictly due to problem with NTv2 files license._

**Comment by DelazJ** [comment]
What do we do here? Close?

**Pull request closed without merging**